enclave-runtime/src/tls_ra/tls_ra_client.rs

```diff
@@ -73,17 +74,20 @@ where
 	///
 	/// We trust here that the server sends us the correct data, as
 	/// we do not have any way to test it.
-	fn read_shard(&mut self) -> EnclaveResult<()> {
```

Comment:
this module really had horrible naming IMHO. Hope this is more understandable now